src/ui/views/meter.rs

```diff
@@ -0,0 +1,196 @@
+use vizia::*;
+
+/// The direction the meter bar shows the peak in.
+/// The semantic is LowToHigh, so DownToUp is the standard vertical meter design
+///
+/// This is also used to decide the orientation of the meter
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Data)]
+pub enum Direction {
+    /// The standard vertical meter direction
+    DownToUp,
```

Comment:
Is there a reason these have to be "x to y" instead of just "y"? Since they're marked as directions, I feel like it's clear that "down" means that the meter goes from top down, and "right" means that the meter goes from left to right.